adcc/adc_pp/modified_transition_moments.py

```diff
@@ -30,47 +30,54 @@
 from adcc.AmplitudeVector import AmplitudeVector
 
 
-def mtm_adc0(mp, dipop, intermediates):
-    return AmplitudeVector(ph=dipop.ov)
+def mtm_adc0(mp, op, intermediates):
+    return AmplitudeVector(ph=op.ov) if op.is_symmetric \
+        else AmplitudeVector(ph=op.vo.transpose())
```

Comment:
If you have to break the one-line if statement it's better to not have a one-line if statement and go for an if-else block in my opnion